### DISCUSSION TITLE
How to install Devika on Cloud VM

### DISCUSSION THREAD

**Discussion by vishalkadam47** [stitionai/devika#620] [Q&A]
I did try in Ubuntu 22.04 everything was good I even get `root: INFO   : Devika is up and running!` 
but at the 7th step ssh from new terminal and 
```
cd ui/
bun install
bun run start
```
its says bun not found even after installing bun.
is there any other way without switching to a new terminal please guide.

**Comment by Polygons1** [comment]
have you tried restarting the shell?
